make timeouts configurable (#508)
vanniktech/gradle-maven-publish-plugin: docs/central.md

Regardless of whether it is done automatically or manually after the staging repository is released the artifacts
will be synced to Maven Central. This process takes 10-30 minutes and when it is completed
the artifacts are available for download.

## Timeouts

From time to time Sonatype tends to time out during staging operations. The default timeouts of the plugin

> are long already but can be modified if needed. The time out for HTTP requests can be modified with

Reviewer: ```suggestion
are long already, but can be modified if needed. The timeout for HTTP requests can be modified with
```

> `SONATYPE_CONNECT_TIMEOUT_SECONDS` which defaults to 1 minute. After a staging repository gets closed

Reviewer: ```suggestion
`SONATYPE_CONNECT_TIMEOUT_SECONDS` which defaults to 1 minute. After a staging repository gets closed,
```

> Sonatype will run several validations on it and the plugin needs to wait for those to finish before it can

Reviewer: ```suggestion
Sonatype will run several validations on it and the plugin needs to wait for those to finish, before it can
```
